docs/root/configuration/http/http_filters/composite_filter.rst

```diff
@@ -0,0 +1,24 @@
+.. _config_http_filters_composite:
+
+Composite Filter
```

Comment:
Without looking at the code, I have some questions:
1) Does it make sense to only support request header matching actions?
2) If we don't support only those actions, how do we make sure the filters actually work?
3) Do we do buffering to replay data if a match requires something beyond request headers?

This may have already been discussed and resulted in the other issue/doc where you talk about a dependency spec for matching, but can we have some more documentation about the limitations we have today? Should we potentially force this filter to only work on request header matching right now?

Reply:
Restricted to request headers (docs updated + tree validator added), so I believe this to be addressed